### DISCUSSION TITLE
username change for access to windows in takecontrol

### DISCUSSION THREAD

**Discussion by thisledome5** [amidaware/tacticalrmm#1865] [General]
Hi, So I noticed that connecting to a remote pc running windows 10 pro that the username that it comes up with is not the user of the windows PC. in fact that userId does not exist on the PC.
In the TRMM screen when you look at the computer listed, it shows the incorrect user, but I do not see anywhere to change this. I read the docmation and browsed discussion groups. but no advice. Does anyone have any pointers on this?

**Comment by thisledome5** [comment]
![Screenshot_13](https://github.com/amidaware/tacticalrmm/assets/45792300/1b832c9d-ca30-46d0-9d0b-9d106762e5c1)
![Screenshot_14](https://github.com/amidaware/tacticalrmm/assets/45792300/aa8001a3-cc15-4b41-a8df-1e5327ee0dcf)
![Screenshot_15](https://github.com/amidaware/tacticalrmm/assets/45792300/13a7bdcb-ea63-48fb-94ed-b0af5289fd4c)

screenshot 13 .this shows the user field in the list of TRMM connected agents. FBC is a correct user of the computer connected to TRMM however when connected to the agent via takecontrol mesh, it shows a user CPELC which does not exist on the computer and there is no w way to change that on the screen. see picture screenshot 14
Screenshot 15 shows the connected agents in mesh but the user field is not correct either. the 3CX and the 3CX Dell units are FBC and CPELC users but don't show there. so no way to change them again.there has to be a way to change the user that logs onto selected computer.

**Reply by dinger1986** [reply]
hello, this is just the desktop of the computer and a mirror image, the user logged in is the same as if you stood infront of the machine.

You could use send command to check `hostname` matches and also use remote background to check it there as well using `hostname` from cmd. Just incase something weird has happened.

**Comment by silversword411** [comment]
You can also see what your username and display name is with
```ps
Get-WmiObject -Class Win32_UserAccount | Select-Object Name, FullName
```
The Name is what you login with. FullName is the display name which can be different

**Reply by thisledome5** [reply]
be that as it may, The FBC computer should be logging in with FBC as there is no user called CPELC on that machine. I checked. So I am not sure where it is coming from.

**Reply by silversword411** [reply]
What is the output of that command?

**Comment by thisledome5** [comment]
It seems that even though the user shown is wrong, the password works anyway which is odd. but all is good.

**Comment by thisledome5** [comment]
It seems that even though the user shown is wrong, the password works anyway which is odd. but all is good.
